pkg/prometheus/operator.go

```diff
@@ -357,15 +354,15 @@ func (c *Operator) reconcile(p *spec.Prometheus) error {
 		// Doing so just based on the deletion event is not reliable, so
 		// we have to garbage collect the controller-created resources in some other way.
 		//
-		// Let's rely on the index key matching that of the created configmap and replica
-		// set for now. This does not work if we delete Prometheus resources as the
+		// Let's rely on the index key matching that of the created configmap and PetSet for now.
```

Comment:
my OCD: I'd say either always line wrap or never... (in vim `gwap` for a whole block/paragraph 😉  )